rest-server/auth-server/src/main/scala/com/actionml/authserver/routes/AuthorizationController.scala

```diff
@@ -3,7 +3,7 @@ package com.actionml.authserver.routes
 import akka.http.scaladsl.model.headers.{BasicHttpCredentials, HttpChallenges}
 import akka.http.scaladsl.server.AuthenticationFailedRejection.{CredentialsMissing, CredentialsRejected}
 import akka.http.scaladsl.server._
-import com.actionml.authserver.model.Client
+import com.actionml.authserver.model.{Client, ClientId}
```

Comment:
Why did you want to call these "Users" I put that into the REST API URIs POST /users/...